src/it/mjaxb-14/pom.xml

```diff
@@ -32,6 +32,14 @@
         <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
     </properties>
 
+    <dependencies>
+        <dependency>
+            <groupId>jakarta.xml.bind</groupId>
+            <artifactId>jakarta.xml.bind-api</artifactId>
+            <version>2.3.2</version>
```

Comment:
This dependency is used in many places. Use a property to control the version in one place?